package.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "wasm-lib",
```

```suggestion
  "name": "@exercism/wasm-lib",
```
We have our own scope!